<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PayDialog</class>
 <widget class="QDialog" name="PayDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>270</width>
     <height>369</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QLabel" name="label_wechatPay">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
    <widget class="QLabel" name="label_aliPay">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_payMsg">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>0</y>
     <width>270</width>
     <height>420</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_pay_drinkName">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>75</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pay_drinkPrice">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_payAmount">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>200</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_pay_commit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>90</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_pay_cancel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>290</y>
      <width>90</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_drinkNameImg">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_drinkPriceImg">
    <property name="geometry">
     <rect>
      <x>43</x>
      <y>130</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_payMsgImg">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>390</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <zorder>label_payMsgImg</zorder>
  <zorder>frame_payMsg</zorder>
  <zorder>tabWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
